src/server/game/Spells/Spell.h

```diff
@@ -274,13 +274,21 @@ class TC_GAME_API SpellCastTargets
         std::string m_strTarget;
 };
 
+enum SpellValueCritFlag : uint8
```

Comment:
Yep P-Kito got me right. It seems that these should be normal values, not flags. My point is to rename enum and values naming. Word "flag" should not be there, cause it is not bitmask.